.circleci/config.yml

```diff
@@ -3,12 +3,12 @@ jobs:
   check:
     docker:
       - image: tiltdev/tilt-extensions-ci:latest
-        
+
     steps:
       - checkout
-      - run: GOBIN=$HOME/.bin go get github.com/tilt-dev/tilt-extensions-ci && go install github.com/tilt-dev/tilt-extensions-ci
+      - run: GOBIN=$HOME/.bin go get github.com/tilt-dev/tilt-extensions-ci && go install 'github.com/tilt-dev/tilt-extensions-ci@latest'
```

Comment:
we should probably change this to use a SHA hash like we do in the main tilt repo -- it makes it a bit easier to make changes atomically

Reply:
Using Docker SHA + Git tag (for `tilt-extensions-ci`) w/ 9029b1d